src/config.ts

```diff
@@ -53,6 +53,19 @@ export type ProviderConfig = {
     /* An optional function to generate the bucket asset key. */
     generateAssetKey?: (filePathOrURL: string, folder: string) => string;
   };
+
+  'cloudflare-r2'?: {
+    endpoint: string;
+    accountId: string;
+    bucket?: string;
+    bucketUrlPublic?: string;
+    accessKeyId?: string;
+    secretAccessKey?: string;
+    apiToken?: string;
+    jurisdiction?: string;
```

Comment:
let's remove jurisdiction and use the endpoint config to keep it more uniform with other providers

Reply:
👍 changed it